autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -79,9 +82,55 @@ def fit(train_data, label, tuning_data=None, output_directory=None, problem_type
             Note: final model returned may be fit on this tuning_data as well as train_data. Do not provide your evaluation test data here!
             In particular, when `num_bagging_folds` > 0 or `stack_ensemble_levels` > 0, models will be trained on both `tuning_data` and `train_data`.
             If `tuning_data = None`, `fit()` will automatically hold out some random validation examples from `train_data`.
-        output_directory : str
+        presets : list or str or dict, default = 'medium_quality_fastest_train'
+            List of preset parameter configurations.
```

Comment:
I think "parameter configurations" is confusing, may say:
"List of configurations that specify how much you care about predictive accuracy, memory-footprint/inference-latency of trained models, and various other properties of the predictor  AutoGluon will return.

Reply:
Added your expanded description. Renamed many references to `parameter` to `arguments`.